### PR TITLE
Milestone 4: Portfolio Website

### PR DESCRIPTION
**Pull request by DannieMo**
This is the fourth milestone of my portfolio website. In this milestone i:
- I used media queries to make my website respond to the different viewport, e.g desktop, and mobile. 
- Added links to my header elements. When you click, it brings you to that section of the website. You should try it out.😃

PS. This is a practice project at Microverse: a remote software programming school. This project is my personal portfolio showing my work, skills, and history. You can seek me out if you are interested in the things I do or just chatting 😉.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
-- This is the third milestone of my portfolio website. In this milestone i:
-- Added contact form and footer 
-- Embedded form spree into my project to be able to send messages.
+- This is the fourth milestone of my portfolio website. In this milestone i:
+- I used media queries to make sure that my website responds to differenr view port, e.g desktop and mobie. 
+- Added links to my header elements. When you click it it brings you to that section of the website. You sohould try it out.😉
 
 PS. This is a practice project at Microverse: a remote software programming school. This project is my personal portfolio showing my work, skills, and history. You can seek me out if you are interested in the things i do or just chatting 😉. 
 
-![image]![image](https://user-images.githubusercontent.com/53879944/126331491-61556a3f-bb2b-468f-9e12-d90eb9d1e11c.png)
```
Comment (emmanuellekamwa) [inline README.md:11]: If I am not mistaken I can see that this image is not displaying itself on your readme. Please you can either fix it or delete it if it is not useful for you guys.

File: index.html [hunk 2]
```diff
@@ -24,148 +24,160 @@
...
-      </section>
+            <form action="https://formspree.io/f/mayadbao" method="POST">
+                <div class="data">
+                    <input type="text" class="contact-element" name="user-f&lname" placeholder="First and Last Name" maxlength="30" required>
+                </div>
+                <label for="email"></label>
+                <input type="email" id="email" class="contact-element1" name="_replyto" placeholder="danielowiri4zoe@gmail.com" required><br>
+                <label for="message"></label>
+                <textarea name="message" id="message" class="contact-element2" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea><br><br>
+                <button type="submit" class="contact-button">Send message</button>
```
Comment (emmanuellekamwa) [inline index.html]: Please try and stick to the design.
The original design 👇🏽

Screenshot 2021-07-22 193145
Your design 👇🏽

Screenshot 2021-07-22 193054
As you can see on the desktop version of the original design the name field is divided into 2 parts but on yours displays what it is supposed to display on mobile view so based on the pictures I have sent you just change the name field and all the placeholders.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @DannieMo and @gabrielcoder247 👋🏽
You have done a great job so far 👏🏽. You have almost passed all the checks necessary but there are a few changes to make in order to have your project approved.

## Required Changes ♻️
_Check the comments under the review._ 👇🏽👀

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**